projects/postgresql/add_fuzzers.diff

```diff
@@ -0,0 +1,68 @@
+diff --git a/src/backend/tcop/postgres.c b/src/backend/tcop/postgres.c
```

Comment:
This is brittle and can block the whole build if the patch no longer applies. Would prefer this upstreamed.

Reply:
The postgreSQL maintainer does not want any changes upstream